### PR TITLE
feat: add explorer button to cellguide and fix global filters

### PR DESCRIPTION
**Pull request by atarashansky**
- Global filters weren't working. A previous PR (#6713) updated the cellguide snapshot to create ontology trees per organism. It also generates valid tissues per organism. This PR updates CellGuide with the new filter behavior as well as adds the explorer instance URLs to the tissue and cell type cards.
- Also, the Tissue Card now has an organism filter for the ontology view and integrated embedding url
- Added a data artifact that captures which explorer CXGs are available per organism, cell type, tissue combination.
- Added a beta chip and tooltip for the new embedding button

## Testing steps

- E2E tests added
- Tested in rdev
- Pipeline unit tests added
- Tested locally

### CODE REVIEW

File: frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx [hunk 1]
```diff
@@ -180,16 +195,62 @@ export default function OntologyDagView({
   // Animation duration - initially zero so the animation doesn't play on load
   const [duration, setDuration] = useState(0);
 
+  // construct explorer URL
+  const formattedSelectedOrganism = selectedOrganism
```
Comment (atarashansky) [inline frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx:199]: This is logic for constructing explorer URL

File: frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx [hunk 2]
```diff
@@ -180,16 +195,62 @@ export default function OntologyDagView({
...
+    tissueId,
+    cellTypeId,
+  ]);
+
   // The raw cell ontology tree data. This is called "rawTree" because it does not contain
   // the "isExpanded" property that is used to track the expanded state of each node, along with
   // other properties like the positions of the nodes.
-  const { data: rawTree } = useCellOntologyTree();
-
+  const { data: rawTree } = useCellOntologyTree(selectedOrganism);
```
Comment (atarashansky) [inline frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx:246]: Ontology trees now must provide the organism name to get the corresponding tree and cell counts.

File: frontend/src/common/queries/cellGuide.ts [hunk 3]
```diff
@@ -85,12 +87,24 @@ async function fetchQuery({
...
  * Generic cell guide hook
  */
-
+// Mapping from organism name to taxon id. Hardcode this for now.
+// This will need to be updated if Census ever includes more organisms.
+export const ORGANISM_NAME_TO_TAXON_ID_MAPPING = {
+  "Homo sapiens": "NCBITaxon_9606",
+  "Mus musculus": "NCBITaxon_10090",
+};
 export function useCellGuideQuery<T = CellGuideResponse>(
```
Comment (tihuan) [inline frontend/src/common/queries/cellGuide.ts:96]: For developer experience and readability, this function has many arguments now, maybe it's time to convert the argument to an object?

E.g.,

```ts
useCellGuideQuery({ dataType, queryId = "", organismName = "", queryLatestSnapshotIdentifier = true}) { ... }
```

**Review by prathapsridharan — APPROVED** [review]
lgtm

File: frontend/src/views/CellGuide/components/CellGuideCard/connect.ts [hunk 4]
```diff
@@ -15,6 +16,8 @@ import { SDSOrgan } from "src/views/CellGuide/components/CellGuideCard/types";
 import { CellType } from "src/views/CellGuide/components/common/OntologyDagView/common/types";
 import { Gene } from "src/views/WheresMyGeneV2/common/types";
 
+// Hardcode this for now
```
Comment (tihuan) [inline frontend/src/views/CellGuide/components/CellGuideCard/connect.ts:19]: what's hard coded here lol?

File: frontend/src/views/CellGuide/components/CellGuideCard/connect.ts [hunk 5]
```diff
@@ -154,7 +155,7 @@ export function useConnect() {
     setGeneInfoGene,
     cellInfoCellType,
     setCellInfoCellType,
-    organismsList,
+    organismsList: Object.keys(ORGANISM_NAME_TO_TAXON_ID_MAPPING),
```
Comment (tihuan) [inline frontend/src/views/CellGuide/components/CellGuideCard/connect.ts:158]: hmm this creates a new object at every render, maybe we should still cache it with `useMemo()` like before?

File: frontend/src/views/CellGuide/components/TissueCard/index.tsx [hunk 6]
```diff
@@ -141,7 +159,7 @@ export default function TissueCard({ description, name }: Props): JSX.Element {
         {/* This prevents auto zooming on the input box on mobile */}
         <meta
           name="viewport"
-          content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1"
```
Comment (tihuan) [inline frontend/src/views/CellGuide/components/TissueCard/index.tsx:144]: for my own education, why is `maximum-scale=1` removed?

File: frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx (hunk 1 above)
Comment (tihuan) [reply frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx:199]: feels like a good opportunity to extract this out to be a standalone function in a `utils.ts` file and import and use it in this index file to keep this main file clean?

File: frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx [hunk 7]
```diff
@@ -398,14 +459,57 @@ export default function OntologyDagView({
   const yMax = height - defaultMargin.top - defaultMargin.bottom;
   const xMax = width - defaultMargin.left - defaultMargin.right;
 
+  const organismText =
+    formattedSelectedOrganism === "homo_sapiens" ? "human" : "mouse";
```
Comment (tihuan) [inline frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx:463]: For context and readability - maybe worth adding a comment on `formattedSelectedOrganism` that the current possible values are `"homo_sapiens" and "mus_musculus". Or perhaps better yet, create a constant to map between value to displayed text like so:

```ts
/**
 * The keys are expected to match the pattern created by `formattedSelectedOrganism` above
 */
const SELECTED_ORGANISM_TO_DISPLAY_TEXT = {
  homo_sapiens: "human",
  mus_musculus: "mouse",
}

organismText = SELECTED_ORGANISM_TO_DISPLAY_TEXT[formattedSelectedOrganism] || "unknown"
```

**Review by tihuan — COMMENTED** [review]
Looks really good for FE! Just some small comments that are non blocking but nice to have 👍 Feel free to address them in a separate PR if this is time sensitive 🕐 

Thank you!

File: frontend/src/views/CellGuide/components/TissueCard/index.tsx (hunk 6 above)
Comment (atarashansky) [reply frontend/src/views/CellGuide/components/TissueCard/index.tsx:144]: I got a linting error:
```
axe-linter (meta-viewport): Ensures <meta name="viewport"> does not disable text scaling and zoomingaxe-linter[meta-viewport](https://dequeuniversity.com/rules/axe/4.8/meta-viewport?application%3Daxe-linter)
```

Comment (atarashansky) [reply frontend/src/views/CellGuide/components/TissueCard/index.tsx:144]: I didn't notice any difference after removing it in mobile view...Not sure what that meta tag is even doing.

File: frontend/tests/features/cellGuide/cellGuide.test.ts [hunk 8]
```diff
@@ -637,9 +639,10 @@ describe("Cell Guide", () => {
           );
         });
 
-        test("Clicks on a cell type node that does NOT have the same tissue in its dropdown in the ontology viewer navigates to the generic cell type page", async ({
+        test.skip("Clicks on a cell type node that does NOT have the same tissue in its dropdown in the ontology viewer navigates to the generic cell type page", async ({
           page,
         }) => {
+          // This test no longer makes sense because there are no cell types that don't have the same tissue in its dropdown anymore.
```
Comment (tihuan) [inline frontend/tests/features/cellGuide/cellGuide.test.ts:645]: @atarashansky should we just delete this test or is there a reason we're skipping it instead? Thanks!

**Pull request merged**